tests/robustness/patches/beforeSendWatchResponse/watch.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/server/etcdserver/api/v3rpc/watch.go b/server/etcdserver/api/v3rpc/watch.go
+index cd834aa..e6aaf2b 100644
+--- a/server/etcdserver/api/v3rpc/watch.go
++++ b/server/etcdserver/api/v3rpc/watch.go
+@@ -460,6 +460,7 @@ func (sws *serverWatchStream) sendLoop() {
+                        sws.mu.RUnlock()
+
+                        var serr error
++                       // gofail: var beforeSendWatchResponse struct{}
```

Comment:
should the failpoint also be backported to 3.5 to ensure the issue is fixed for the latest 3.5 release?

Reply:
Right, there is even a issue for that https://github.com/etcd-io/etcd/issues/17817